lib/puppetserver/ca/action/list.rb

```diff
@@ -57,9 +61,16 @@ def run(input)
           config = input['config']
           certnames = input['certname'] || []
           all = input['all']
+          output_format = input['format'] || "text"
+          filter_names = []
```

Comment:
I think we can get rid of this too, for the same reason as not needing to set `exit_code` up here. Leaving it is a little confusing, since it gets defined as a lambda below, but this implies that it should be an array.